### ISSUE TITLE
Component crashes with not valid angles

### ISSUE THREAD

**Issue by alcantara09** [Brazilian-Institute-of-Robotics/simulation-orogen-imaging_sonar_simulation#16]
When the component starts without configuring the start, step and end angles, it initializes the three parameters with NaNs and when trying to write a pose, the component crashes. 

It would be nice to have a validation of the configuration before starting the component.

**Comment by romulogcerqueira** [comment]
Hi @alcantara09, I will take care if this. 😉

**Comment by romulogcerqueira** [comment]
@alcantara09, could you test this fix (https://github.com/Brazilian-Institute-of-Robotics/simulation-orogen-imaging_sonar_simulation/pull/17)?

**Comment by romulogcerqueira** [comment]
This problem was already solved. I am closing this issue.
